<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogUpdatePassAdmin</class>
 <widget class="QDialog" name="dialogUpdatePassAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Update password</string>
  </property>
  <widget class="QGroupBox" name="groupBoxUpdatePass">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>531</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Update password</string>
   </property>
   <widget class="QLabel" name="labelRepeatNewPasswd">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Repeat new password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNewPasswd">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>New password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOldPasswd">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Old password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOldPass">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNewPass">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNewPassRepeat">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelError">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelErrorOldPass">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnUpdatePasswd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Update password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
